### DISCUSSION TITLE
Sorting & filtering

### DISCUSSION THREAD

**Discussion by naftis** [uppi-fi/uppi.fi#47] [General]
How should we design them?

**Comment by naftis** [comment]
Github Gist is a similar service than ours but for text. When we implement sorting and/or paging in the backend, the Github styled sort & type-buttons could be useful?
![image](https://user-images.githubusercontent.com/1322608/138005572-8bddc046-67e7-4273-ac3c-80985a72835b.png)

**Reply by eioo** [reply]
I like this 👍 Current solution is bit hard to understand
